diff --git a/common/iobuf.h b/common/iobuf.h
index 9c9650c61..a3d9bd547 100644
--- a/common/iobuf.h
+++ b/common/iobuf.h
@@ -249,14 +249,7 @@ struct iobuf_struct
   int subno;
 };

-#ifndef EXTERN_UNLESS_MAIN_MODULE
-#if defined (__riscos__) && !defined (INCLUDED_BY_MAIN_MODULE)
-#define EXTERN_UNLESS_MAIN_MODULE extern
-#else
-#define EXTERN_UNLESS_MAIN_MODULE
-#endif
-#endif
-EXTERN_UNLESS_MAIN_MODULE int iobuf_debug_mode;
+extern int iobuf_debug_mode;


 /* Change the default size for all IOBUFs to KILOBYTE.  This needs to
